plugins/template/test/TemplateCode_unittest.cpp

```diff
@@ -0,0 +1,38 @@
+#include "TemplateCode.h"
```

Comment:
nit: 
```suggestion
#include <TemplateCode.h>
```
because it's not actually in the same directory.